dm/dm/config/task.go

```diff
@@ -1112,6 +1131,15 @@ func NewSyncerConfigsForDowngrade(syncerConfigs map[string]*SyncerConfig) map[st
 	return syncerConfigsForDowngrade
 }
 
+// omitDefaultVals change default value to empty value for new config item.
+// If any default value for new config item is not empty(0 or false or nil),
+// we should change it to empty.
+func (c *SyncerConfigForDowngrade) omitDefaultVals() {
+	if c.SafeModeDuration == DefaultSyncerConfig().SafeModeDuration {
```

Comment:
prefer
```
const safeModeDuration
```
rather than define it in DefaultSyncerConfig

Reply:
The consideration is define it in DefaultSyncerConfig can follow `CheckpointFlushInterval` directly.

Reply:
You can directly define it as
```
var (
    SafeModeDuration = strconv.Itoa(2*defaultCheckpointFlushInterval) + "s"
)
```